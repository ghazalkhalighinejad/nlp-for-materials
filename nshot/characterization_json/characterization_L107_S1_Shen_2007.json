{"Transmission_Electron_Microscopy": {"Description": "Equipment not specified."}, "Dielectric_and_Impedance_Spectroscopy_Analysis": {"Equipment": "HP 4194A impedance analyzer"}, "XRay_Diffraction_and_Scattering": {"Description": "Equipment not specified."}, "Raman_Spectroscopy": {"Description": "Equipment not specified."}}