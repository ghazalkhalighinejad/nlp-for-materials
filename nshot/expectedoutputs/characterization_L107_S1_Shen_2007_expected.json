{
    "microscopy": {
        "transmission_electron_microscopy": {
            "Description": "Equipment not specified."
        },
        "scanning_electron_microscopy": {},
        "atomic_force_microscopy": {},
        "optical_microscopy": {}
    },
    "spectroscopy": {
        "fourier_transform_infrared_spectroscopy": {},
        "xray_photoelectron_spectroscopy": {},
        "nuclear_magnetic_resonance": {},
        "raman_spectroscopy": {
            "Description": "Equipment not specified."
        },
        "dielectric_and_impedance_spectroscopy_analysis": {
            "Equipment": "HP 4194A impedance analyzer"
        }
    },
    "thermochemical": {
        "differential_scanning_calorimetry": {},
        "thermogravimetric_analysis": {},
        "dynamic_mechanical_analysis": {}
    },
    "scattering_and_diffraction": {
        "xray_diffraction_and_scattering": {
            "Description": "Equipment not specified."
        }
    },
    "others": {
        "pulsed_electro_acoustic": {},
        "rheometry": {},
        "electrometry": {}
    }
}
